src/App.jsx

```diff
@@ -4,9 +4,31 @@ import AddTodoForm from "./components/AddTodoForm";
 import "./App.css";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 
+function sortTodoByCreatedTimeAsc(objectA, objectB) {
+  return new Date(objectA.createdTime) - new Date(objectB.createdTime);
+}
+
+function sortTodoByCreatedTimeDesc(objectA, objectB) {
+  return new Date(objectB.createdTime) - new Date(objectA.createdTime);
+}
+
+function sortTodoByTitleAsc(objectA, objectB) {
+  if (objectA.title < objectB.title) return -1;
+  if (objectA.title > objectB.title) return 1;
+  return 0;
+}
+
+function sortTodoByTitleDesc(objectA, objectB) {
+  if (objectA.title < objectB.title) return 1;
+  if (objectA.title > objectB.title) return -1;
+  return 0;
+}
+
```

Comment:
It's good that these are outside of the component - it means React won't have to re-initialize these function each time the component re-renders. One thing to consider for the future (not now or final project requirements) is refactoring these out into a utilities directory.